addon/-private/cache.ts

```diff
@@ -46,6 +47,7 @@ const { assert, deprecate } = Orbit;
 
 export interface CacheSettings {
   sourceCache: MemoryCache;
+  store: Store;
```

Comment:
I'd prefer to make `store` optional to allow for forked caches to exist that are not directly associated with a `Store` (to mirror what's possible with `MemoryCache`):

```suggestion
  store?: Store;
```